templates/repo/issue/milestones.tmpl

```diff
@@ -64,6 +64,7 @@
 						<span class="issue-stats">
 							<i class="octicon octicon-issue-opened"></i> {{$.i18n.Tr "repo.issues.open_tab" .NumOpenIssues}}
 							<i class="octicon octicon-issue-closed"></i> {{$.i18n.Tr "repo.issues.close_tab" .NumClosedIssues}}
+							{{if .TotalTrackedTime}}<i class="octicon octicon-clock"></i> {{.TotalTrackedTime|Sec2Time}}{{end}}
```

Comment:
No, because `.TotalTrackedTime` is zero if timetracker is disabled because `TotalTrackedTime` is only loaded if it is enabled. (https://github.com/go-gitea/gitea/pull/3341/files#diff-ece22e0fe138da6e0fe0f25213406313R1142)

Reply:
@lafriks Done